migrator/migrations/n_04_to_n_05_postgres_images/migration.go

```diff
@@ -0,0 +1,87 @@
+// Code originally generated by pg-bindings generator.
+
+package n4ton5
+
+import (
+	"context"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations"
+	"github.com/stackrox/rox/migrator/migrations/loghelper"
+	legacy "github.com/stackrox/rox/migrator/migrations/n_04_to_n_05_postgres_images/legacy"
+	pgStore "github.com/stackrox/rox/migrator/migrations/n_04_to_n_05_postgres_images/postgres"
+	store "github.com/stackrox/rox/migrator/migrations/n_04_to_n_05_postgres_images/store"
+	"github.com/stackrox/rox/migrator/types"
+	rawDackbox "github.com/stackrox/rox/pkg/dackbox/raw"
+	pkgMigrations "github.com/stackrox/rox/pkg/migrations"
+	pkgSchema "github.com/stackrox/rox/pkg/postgres/schema"
+	"github.com/stackrox/rox/pkg/sac"
+	"gorm.io/gorm"
+)
+
+var (
+	migration = types.Migration{
+		StartingSeqNum: pkgMigrations.CurrentDBVersionSeqNumWithoutPostgres() + 4,
+		VersionAfter:   storage.Version{SeqNum: int32(pkgMigrations.CurrentDBVersionSeqNumWithoutPostgres()) + 5},
```

Comment:
How did we calculate `+4` and `+5`?

Reply:
The number is derived from the --migration-seq, giving the tables a sequence.